src/pages/index.tsx

```diff
@@ -196,20 +198,26 @@ const HomePage = (): jsx.JSX.Element => (
               work today, in whichever way suits you best.
             </p>
           </InnerText>
-          <h3 css={[readerFundedHeadingCss, supportReaderFundedHeadingCss]}>Support the Guardian</h3>
+          <h3 css={[readerFundedHeadingCss, supportReaderFundedHeadingCss]}>
+            Support the Guardian
+          </h3>
           <ReaderFundedContributeCard />
-          <h3 css={[readerFundedHeadingCss, printReaderFundedHeadingCss]}>Print Subscriptions</h3>
+          <h3 css={[readerFundedHeadingCss, printReaderFundedHeadingCss]}>
+            Print Subscriptions
+          </h3>
           <div css={readerFundedSubscribeCardHolderCss}>
             <ReaderFundedSubscribeCard
               title="Newspaper"
               bodyText="Convenient and money-saving, get a newspaper delivered to your door, or pick it up from your local shop. Choose your subscription, from daily to weekend-only."
               href="https://support.theguardian.com/subscribe/paper"
-              imagePath="/about/images/newspaper-desktop.png" />
+              imagePath="/about/images/newspaper-desktop.png"
```

Comment:
something for another time, but I noticed these images are huge (720kb, 1844x520px) for the size they're displayed at